cli/tests/sync_application_add.sh

```
@@ -0,0 +1,98 @@
+find sync-test-node-application/src/modules/lib-dummy sync-test-node-application/src/modules/svc-dummy sync-test-node-application/src/modules/wrk-dummy -type f \
+  -exec sed -i.bak 's/@sync-test-dummy-application\//@sync-test-node-application\//g' {} \;
+find sync-test-node-application/src/modules -name "*.bak" -type f -delete
+
+# Run sync to add the new projects
+echo "[INFO] Running sync all to add new projects"
+PROMPTS_JSON='{"svc-dummy": {"category": "service", "database": "postgresql", "infrastructure": "none", "description": "Dummy service"}, "lib-dummy": {"category": "library", "description": "Dummy library"}, "wrk-dummy": {"category": "worker", "type": "database", "database": "postgresql", "description": "Dummy worker"}}'
+
+RUST_BACKTRACE=1 cargo run --release sync all -p sync-test-node-application -c -P "$PROMPTS_JSON"
+
+# Verify build
+cd sync-test-node-application/src/modules
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)